.travis.yml

```diff
@@ -21,7 +23,7 @@ addons:
   apt:
     packages:
       - graphviz  # for building docs
-      - libreadline6-dev  # epics build deps
+      - libreadline6-dev  # epics deps
```

Comment:
There is way to patch the build configuration to not need readline (see https://github.com/NSLS-II/lightsource2-recipes/blob/master/recipes-tag/epics-base/config_readline.diff )  Might be worth doing?

Reply:
@tacaswell I'd happily accept a PR over at klauer/epics-on-travis!

Reply:
I'm opened an issue there and tagged Tom. :-D